### PR TITLE
Cloud spring cleaning 2024 04 30

### PR DESCRIPTION
**Pull request by CloudEllie**
- Updated Validator page with additional context, as posted in the [org repo discussion](https://github.com/code-423n4/org/issues/159#issuecomment-2085243965)
- Updated submission guidelines, awarding info, and judging guidelines per the [new rules](https://code4rena.com/blog/code4rena-spring-update-2024):
   - Hunter / Gatherer bonuses
   - QA reports: new Governance section, discouragement of non-criticals
- Remove references to bot reports and Analyses
- Clarify that gas pools are now optional

### CODE REVIEW

File: roles/wardens/submission-guidelines.md [hunk 1]
```diff
@@ -23,57 +23,32 @@ It is also recommended to ensure you receive email confirmation of each submissi
...
-2. Architecture feedback
-3. Centralization risks
-4. Systemic risks
-5. Other recommendations
-6. How much time did you spend?
-
-### QA reports (low/non-critical)
+### QA reports (low/governance)
 
 Low and non-critical findings must be submitted as a single QA report per warden. We allocate a **fixed 2.5% of prize pools toward QA reports.**
```
Comment (itsmetechjay) [inline roles/wardens/submission-guidelines.md:34]: I think this should be updated to 4%.

Comment (CloudEllie) [reply roles/wardens/submission-guidelines.md]: TY - corrected [here](https://github.com/code-423n4/docs/pull/97/commits/89eaa118d46fdfd2b35b2e635fe1d20168bc81a2)

File: roles/certified-contributors/README.md [hunk 2]
```diff
@@ -5,15 +5,15 @@ description: In order to create opportunities for contributions which rely on es
 
 Contributors who have provided ID verification and a signed agreement may be eligible to participate in:
 
-- Private or invite-only contests
-- Scout role (focused on scoping and pre-contest code intel)
+- Private or invite-only audits
+- Scout role (focused on scoping and pre-audit code intel)
 - [Judging](/roles/judges/README.md)
-- ["Backstage" warden opportunities](backstage-wardens.md) (post-contest triage and post-judging QA)
+- [SR opportunities](sr-backstage-wardens.md) (post-audit triage and post-judging QA)
```
Comment (liveactionllama) [inline roles/certified-contributors/README.md]: I think `post-audit triage` should perhaps be removed completely here?

File: roles/certified-contributors/sr-backstage-wardens.md [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
-- Immediate access to findings repos after audits conclude
+- Findings repos, immediately after audits conclude
 - Post-judging QA
 
-The minimum criteria to become +Backstage are as follows: 
+The minimum criteria to become a Certified SR are as follows: 
 
 1. Be approved as a Certified C4 contributor;
 1. Submit valid findings to at least 3 Code4rena audits (i.e. valid findings on the [Code4rena leaderboard](https://code4rena.com/leaderboard/));
 1. Have at least 1 high severity finding OR 3 medium severity findings on the [Code4rena leaderboard](https://code4rena.com/leaderboard/), OR score A on a QA report, Gas report, or Analysis (formerly "Advanced Analysis"; basic Analysis grades are not eligible);
```
Comment (liveactionllama) [inline roles/certified-contributors/sr-backstage-wardens.md:12]: Are changes needed here related to grades and/or analyses?

File: roles/certified-contributors/validators.md [hunk 4]
```diff
@@ -3,14 +3,14 @@
 > Validators decentralize triage by reviewing submissions from wardens with accuracy rates below the qualifying threshold (eg 50%).
 > 
 
-All open competitive audits at Code4rena that begin on or after May 1, 2024 will include Validators.
+All open competitive audits at Code4rena that begin on or after April 30, 2024 will include Validators.
 
 ## Validator tl;dr
 
 - Each competition has a **qualifying threshold** that allows wardens to bypass validators. This threshold is based on your submission accuracy rate and being established as a quality contributor and as non-sybil. (For those familiar with ‘backstage warden’ criteria, it is essentially that plus an acceptable accuracy rate.)
```
Comment (liveactionllama) [inline roles/certified-contributors/validators.md:10]: Maybe slightly tweak this sentence to something like this instead?

> This threshold is based on your submission accuracy rate as well as being established as a quality contributor and as non-sybil.

File: roles/certified-contributors/sr-backstage-wardens.md (hunk 3 above)
Comment (CloudEllie) [reply roles/certified-contributors/sr-backstage-wardens.md:12]: Not yet, since the new approach is not yet visible on the leaderboard.

**Pull request merged**